apps/judicial-system/web/src/routes/Court/Indictments/Advocates/Advocates.tsx

```diff
@@ -28,18 +26,17 @@ const Advocates = () => {
   const { workingCase, isLoadingWorkingCase, caseNotFound } =
     useContext(FormContext)
   const router = useRouter()
-  const { sendNotification, isSendingNotification } = useCase()
+
   const { formatMessage } = useIntl()
 
   const handleNavigationTo = useCallback(
     async (destination: string) => {
```

Comment:
Yes agreed, I'm handling the civil claimants logic on another branch where I'll be handling all that

Reply:
I'll actually do the removing of the roles rules here though actually, and then just handle the sending of civil claimant notifications on the other branch. There aren't ever civil claimants on indictment cases so it shouldn't matter too much but the changes will still go out together instead of waiting to handle civil claimants until later since we're in this zone.